src/components/sticky-object.js

```diff
@@ -51,22 +51,22 @@ AFRAME.registerComponent("sticky-object", {
   _onRelease() {
     // Happens if the object is still being held by another hand
     if (this.el.is("grabbed")) return;
-
     if (
       this.data.autoLockOnRelease &&
       (this.data.autoLockSpeedLimit === 0 ||
-        this.el.body.velocity.lengthSquared() < this.data.autoLockSpeedLimit * this.data.autoLockSpeedLimit)
+        this.el.components["ammo-body"].getVelocity() < this.data.autoLockSpeedLimit)
     ) {
       this.setLocked(true);
     }
-    this.el.body.collisionResponse = true;
+
+    this.el.setAttribute("ammo-body", { collisionFilterGroup: 1 });
   },
 
   _onGrab() {
     if (!this.el.components.grabbable || this.el.components.grabbable.data.maxGrabbers === 0) return;
 
+    this.el.setAttribute("ammo-body", { collisionFilterGroup: this.locked ? 2 : 1 });
```

Comment:
I suggest naming and exporting these filter groups as named constants rather than refer to them by their numerical value. (Except in the html, where we can't avoid it).